src/utils/projectTypes.ts

```diff
@@ -0,0 +1,28 @@
+import { getValidSchemaDictionary } from "./digitalPlanningSchema";
+
+export function formatRawProjectTypes(rawProjectTypes: string[]): string {
+  const schemaProjectTypes = getValidSchemaDictionary("ProjectType");
+  if (schemaProjectTypes) {
+    const matchingDescriptions = Object.entries(schemaProjectTypes)
+      .filter(([key, _description]) => rawProjectTypes.includes(key))
+      .map(([_key, description]) => description.toLocaleLowerCase());
+    if (matchingDescriptions.length === 0) {
+      return "Unknown";
+    }
+    return formatHumanReadableProjectTypes(matchingDescriptions);
+  } else {
+    return "Unknown";
+  }
+}
+
+export function formatHumanReadableProjectTypes(
+  projectTypes: string[],
+): string {
+  // Join in readable format - en-US ensures we use Oxford commas
+  const formatter = new Intl.ListFormat("en-US", { type: "conjunction" });
```

Comment:
It would be nice to remove or polyfill this whilst we're touching this function 🙏

Reply:
Good call - will pick up !

Reply:
Now updated using this solution: https://formatjs.io/docs/polyfills/intl-listformat/